scripts/setupDegradedTests.sh

```diff
@@ -16,7 +32,7 @@ uaac curl -X POST /identity-zones -H 'Content-Type: application/json' -d'{ "id":
 uaac -t curl -H "X-Identity-Zone-Id:test-saml-zone" -XPOST -H"Content-Type:application/json" -H"Accept:application/json" --data '{ "client_id" : "admin", "client_secret" : "adminsecret", "scope" : ["uaa.none"], "resource_ids" : ["none"], "authorities" : ["uaa.admin","clients.read","clients.write","clients.secret","scim.read","scim.write","clients.admin", "sps.write", "sps.read", "zones.test-saml-zone.admin", "idps.read", "idps.write"], "authorized_grant_types" : ["client_credentials"]}' /oauth/clients
 
 #Login to test-saml-zone
-uaac target http://test-saml-zone.localhost:8080/uaa
+uaac target ${PROTOCOL}://test-saml-zone.$PUBLISHED_DOMAIN
```

Comment:
When creating admin clients for zones, parameterize the secrets and set them in uaa-cf-release.